tests/migration_test.go

```diff
@@ -1616,6 +1616,56 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 					tests.CleanNodes()
 				})
 
+				It("should migrate a VMI only one time", func() {
+					tests.SkipIfVersionBelow("Eviction of completed pods requires v1.13 and above", "1.13")
+
+					vmi = fedoraVMIWithEvictionStrategy()
+
+					By("Starting the VirtualMachineInstance")
+					vmi = runVMIAndExpectLaunch(vmi, 180)
+
+					tests.WaitAgentConnected(virtClient, vmi)
+
+					// Mark the masters as schedulable so we can migrate there
+					setMastersUnschedulable(false)
```

Comment:
Why do you have to do that?

Reply:
This is a lesson from stabilization efforts that we were doing to the migration tests. Because we have only 1 master on the test environment, we first ensure that it will not be the node that we drain by tainting it. Then, we need to remove the taint to allow the VM to migrate there.

Reply:
This might actually cause unexpected results. In some clusters I'm aware of masters don't have /dev/kvm (e.g. nested virt is not enabled) so can't be scheduled on even with this approach.

Reply:
@stu-gott if the master doesn't have /dev/kvm so in order to execute the migration tests you need another node anyway.
But how would it cause unexpected results? The only thing that is done here is ensuring that we don't drain one of the masters.

Reply:
I assumed passing false to this function would render the master schedulable? If I'm wrong, then nevermind.

Reply:
I think we should stop doing real "drains" in our test suite. Repeatedly calling the eviction endpoint until the virt-launcher pod is gone and no other migration gets triggered should do. Full "drain" is in my oppinion too error prone on such small clusters.